Source/Core/VideoBackends/OGL/OGLTexture.cpp

```diff
@@ -442,11 +440,12 @@ void OGLStagingTexture::CopyFromTexture(const AbstractTexture* src,
       // TODO: Investigate whether it's faster to use glReadPixels() with a framebuffer, since we're
       // copying the whole texture, which may waste bandwidth. So we're trading CPU work in creating
       // the framebuffer for GPU work in copying potentially redundant texels.
+      glActiveTexture(GL_TEXTURE9);
+      glBindTexture(GL_TEXTURE_2D_ARRAY, gltex->GetRawTexIdentifier());
```

Comment:
Shouldn't we use gltex->GetRawTexIdentifier() here as well?

Reply:
AFAICT glGetTexImage takes a target, not name, as the first parameter, which is the incorrect assumption I made for glGetTextureSubImage.